collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -2297,7 +2301,7 @@ public void denied() {
                         formController.setLanguage(newLanguage);
                     } catch (Exception e) {
                         // if somehow we end up with a bad language, set it to the default
-                        Timber.e("Ended up with a bad language. %s", newLanguage);
+                        Timber.e(e, "Ended up with a bad language. %s", newLanguage);
```

Comment:
I see this in a lot of our logs and I can't tell if it's actually meaningful. Hopefully seeing the exception will give us some insights. I'm hoping it turns out to be expected and we can just remove the logging.

Reply:
It takes place every time I open the demo form. I think it's caused by language code, we have just label but it should be label::English (en) https://xlsform.org/en/#multiple-language-support

Reply:
Yeah, I saw the stacktrace locally and realized it's really nothing so I should take this change out. 

It doesn't have to do with language codes because those are ignored in JR/Collect for now. I think it's just that because no language was explicitly chosen from the menu the database column for language has a null value.

Reply:
To be explicit, I think it's important to clarify/clean up this logging because it can waste our time when troubleshooting logs on Crashlytics.